modules/idlgen/plugin/src/main/scala/IdlGenPlugin.scala

```diff
@@ -108,7 +114,8 @@ object IdlGenPlugin extends AutoPlugin {
     srcGenSourceDirs := Seq(srcGenSourceDir.value),
     srcGenIDLTargetDir := (Compile / resourceManaged).value / idlType.value,
     srcGenTargetDir := (Compile / sourceManaged).value,
-    genOptions := Seq.empty
+    genOptions := Seq.empty,
+    idlGenCodecImports := List(BigDecimalAvroMarshallers, JavaTimeDateAvroMarshallers)
```

Comment:
Are we tying the default values to Avro? Do you think we could the `idlType` setting to figure out these values?